2019Labs/CustomSecurityContent/documentation/lab0_setup-devconf.adoc

```diff
@@ -0,0 +1,114 @@
+:toc2:
+:linkattrs:
+:imagesdir: images
+
+= Lab Exercise 0: Setup Steps
+
+
+== 0.1 Accessing Your Dedicated Lab Environment Using Your Provided GUID
+
+. From your laptop, navigate to the https://www.opentlc.com/gg/gg.cgi?profile=generic_matyc[*Lab GUID Assignment page*^]. Firefox with plugins disabled recommended.
```

Comment:
The only difference in this file from [lab0_setup-rhte.adoc](https://github.com/RedHatDemos/SecurityDemos/blob/master/2019Labs/CustomSecurityContent/documentation/lab0_setup-rhte.adoc) is that we changed the URL to use the profile `generic_matyc`

Reply:
Plus, we would like to add instructions for people that use the local Fedora VM. I recall
- run playbooks
- in your head, swap rhel8 with fedora when following instructions